packages/ckeditor5-html-support/src/utils.ts

```diff
@@ -221,3 +221,18 @@ export function toPascalCase( data: string ): string {
 export function getHtmlAttributeName( viewElementName: string ): string {
 	return `html${ toPascalCase( viewElementName ) }Attributes`;
 }
+
+/**
+ * Checks if the given attribute name is a GHS attribute name.
+ *
+ * @param name Attribute name to check.
+ * @returns `true` if the attribute name is a GHS attribute name, `false` otherwise.
+ * @example
+ * ```js
+ * isGHSAttributeName( 'htmlPAttributes' ); // true
+ * isGHSAttributeName( 'span' ); // false
+ * ```
```

Comment:
I can't find any other place in our code with the `example` tag used.

```suggestion
 * Checks if the given attribute name is a GHS attribute name.
 *
 * ```ts
 * isGHSAttributeName( 'htmlPAttributes' ); // true
 * isGHSAttributeName( 'span' ); // false
 * ```
 *
 * @param name Attribute name to check.
 * @returns `true` if the attribute name is a GHS attribute name, `false` otherwise.
```